magpie-aws/src/main/java/io/openraven/magpie/plugins/aws/discovery/AWSUtils.java

```diff
@@ -207,6 +225,28 @@ public static GetMetricStatisticsResponse getCloudwatchMetricStatistics(String r
     }
   }
 
+  public static GetMetricStatisticsResponse getStaleDataCloudwatchMetrics(String regionID, String namespace, String metric, Statistic statistic, List<Dimension> dimensions, MagpieAWSClientCreator clientCreator) {
+
+    try (final var client = clientCreator.apply(CloudWatchClient.builder()).region(Region.of(regionID)).build()) {
+
+      // The start time is t-minus 2 days (48 hours) because an asset is considered "active" if it's been updated within
```

Comment:
Let's remove the 2 day comment, as Magpie doesn't have a concept of active vs inactive resources, it simply discovers what exists.